src/util/WalletBackendAdapter.ts

```diff
@@ -120,13 +125,20 @@ export const makeWalletBridgeFromFollowers = (
   currentFollower: ValueFollower<CurrentWalletRecord>,
   updateFollower: ValueFollower<UpdateRecord>,
   beansOwingFollower: ValueFollower<string>,
+  vbankAssetsFollower: ValueFollower<unknown>,
+  agoricBrandsFollower: ValueFollower<unknown>,
```

Comment:
please include the types. if they can't be imported just write out what this code is assuming.

Reply:
also this is getting very long for a positional arguments list. very much worth turning into a named options argument, though not necessarily this PR

Reply:
Added types here and a bit below.

Reply:
I was going to say something about the `unknown` type, but if it typechecks (without casts), that means the code doesn't care what the type is, right? i.e. the code is assuming nothing.